src/main/java/org/sagebionetworks/bridge/services/ParticipantService.java

```diff
@@ -824,6 +845,44 @@ public StudyParticipant updateIdentifiers(App app, CriteriaContext context, Iden
         // return updated StudyParticipant to update and return session
         return getParticipant(app, account.getId(), false);
     }
+
+    public void downloadParticipantRoster(String appId, String userId, String password, String studyId) throws JsonProcessingException {
+        // password must contain at least 8 characters, 1 upper case, 1 lower case, and 1 number
+        checkPassword(password);
+
+        ObjectMapper jsonObjectMapper = new ObjectMapper();
+
+        // wrap message as nested json node
+        ObjectNode requestNode = jsonObjectMapper.createObjectNode();
+        requestNode.put(REQUEST_KEY_APP_ID, appId);
+        requestNode.put(REQUEST_KEY_USER_ID, userId);
+        requestNode.put(REQUEST_KEY_PASSWORD, password);
+        requestNode.put(REQUEST_KEY_STUDY_ID, studyId);
+
+        ObjectNode requestMsg = jsonObjectMapper.createObjectNode();
+        requestMsg.put(REQUEST_KEY_SERVICE, DOWNLOAD_ROSTER_SERVICE_TITLE);
+        requestMsg.set(REQUEST_KEY_BODY, requestNode);
+
+        String requestJson = jsonObjectMapper.writeValueAsString(requestMsg);
+
+        // sent to SQS
+        String queueUrl = bridgeConfig.getProperty(CONFIG_KEY_DOWNLOAD_ROSTER_SQS_URL); //TODO ask about the url
+        SendMessageResult sqsResult = sqsClient.sendMessage(queueUrl, requestJson);
+        LOG.info("Sent request to SQS for userId=" + userId + ", app=" + appId +
+                "; receipted message ID=" + sqsResult.getMessageId());
+    }
+
+    private void checkPassword(String password) {
+        String regex = "^(?=.*[a-z])(?=.*[A-Z])(?=.*\\d)[a-zA-Z\\d]{8,}$";
+
+        Pattern pattern = Pattern.compile(regex);
+        Matcher matcher = pattern.matcher(password);
+
+        if (!matcher.matches()) {
+            throw new BadRequestException("Password must contain at least 8 characters: " +
+                    "1 upper case letter, 1 lower case letter, and 1 number.");
+        }
+    }
```

Comment:
Consider using ValidateUtils.validatePassword(). See for example https://github.com/Sage-Bionetworks/BridgeServer2/blob/develop/src/main/java/org/sagebionetworks/bridge/validators/PasswordResetValidator.java#L45

You might want to pass in the ParticipantRosterRequest into the service and make a ParticipantRosterRequestValidator. See for example https://github.com/Sage-Bionetworks/BridgeServer2/blob/develop/src/main/java/org/sagebionetworks/bridge/services/AuthenticationService.java#L395